src/top/mod.rs

```diff
@@ -12,18 +12,20 @@ use url::Url;
 #[command(rename_all = "kebab-case")]
 pub struct Opts {
     /// Interval to sample metrics at, in milliseconds
-    #[arg(default_value = "500", short = 'i', long)]
+    #[arg(default_value = "1000", short = 'i', long)]
     interval: u32,
 
     /// Vector GraphQL API server endpoint
     #[arg(short, long)]
     url: Option<Url>,
 
     /// Humanize metrics, using numeric suffixes - e.g. 1,100 = 1.10 k, 1,000,000 = 1.00 M
-    #[arg(short = 'H', long)]
+    #[arg(short = 'H', long, default_value_t = true)]
```

Comment:
This just makes sense, I think.

The throughput rates already indicate change so you really don't need to see that you've gone from 7,267,133 events to 7,267,134 to understand events are flowing, etc.